index.html

```diff
@@ -244,22 +248,22 @@ <h2 class="title myself">About<br>Myself</h2>
           <h2 class="title cnt-form">Contact me</h2>
           <div class="container-form">
             <p class="cnt-text">If you have an application you are interested in developing, a feature that you need
-              built
-              or a project that needs coding. I’d love to help with it
+              built or a project that needs coding. I’d love to help with it
             </p>
             <form class="cnt-form-controls" action="https://formspree.io/f/xleakzoe" method="post">
-              <label for="name">
-                <input class="form-field" type="text" id="name" name="user_name" placeholder="Enter your name" required
-                  maxlength="30">
+              <label for="name"><span class="label-text">Please enter your name<span class="red">*</span></span>
+                <input class="form-field" type="text" id="name" name="user_name" placeholder="Enter your name here"
+                  required maxlength="30">
               </label>
-              <label for="email">
-                <input class="form-field" type="email" id="email" name="user_email" placeholder="Enter your email"
+              <label class="label-text" for="email">Please enter your email<span class="red">*</span>
+                <input class="form-field" type="email" id="email" name="user_email" placeholder="Enter your email here"
                   required>
               </label>
-              <label for="msg">
+              <label id="last" class="label-text" for="msg">Please write your message<span class="red">*</span>
```

Comment:
[OPTIONAL] Good job adding labels to your inputs so they're accessible to everyone. I think you could hide these to the average user and make them accessible only to screen readers. [Read more about it here.](https://css-tricks.com/inclusively-hidden/) It will help you match the design in Figma while also keeping your web-accessible.

Reply:
I see your point. If I remove the labels it will not pass the Dev Tools toolbox test, because it will say there are empty labels. That is why I put them. I am really confused and frustrated because there is one template and requirements and now you are asking me to make something else, and either way, I will be punishable. If I make it as the Figma template without labels, somebody could reject my pull request, because one of the requirements of the accessibility step is not fulfilled (interaction: https://www.w3.org/WAI/test-evaluate/preliminary/#interaction). So, when I made this part to fulfill the step now I am against the Figma design, which does not require labels. I will not introduce this change, since it is optional and I need more clarity about what to follow and what is more important.

Reply:
Hello, I'm sorry my explanation wasn't clear enough. I wasn't asking you to remove the labels from your HTML document. I was asking you to hide them from plain sight through a class in CSS. This is usually a class call `.screen-readers-only`.

The screen readers will read them since they're present in the HTML file, but for users not using screen readers they won't be present and the design will remain intact.

That's why the name of the article is "Inclusively Hidden". Also, I marked this as optional and optional requirements won't stop a project from being approved.

I hope you find this insightful and you can get my suggestion. If you have any doubts, don't hesitate to reach to me on Slack.

Reply:
@D4nielJ Thanks for your clarification. Today we are starting with Bootstrap and it will be a good opportunity to implement this class in my future design.

Reply:
In Bootstrap they have implementations ready to go. So you don't even need to add them to your stylesheets. [Check them out](https://getbootstrap.com/docs/5.1/helpers/visually-hidden/). 😉